test/modules/parser/parsers/parse-xviz-stream.spec.js

```diff
@@ -314,7 +335,8 @@ tape('parseXVIZStream#primitive no-data entries', t => {
       pointCloud: null,
       images: [],
       components: [],
-      time: 100.5425
+      time: 100.5425,
+      __metadata: {category: 'PRIMITIVE', primitive_type: 'polygon2d'}
```

Comment:
`polygon2d` is only xviz v1, not sure if this makes sense outside of v2.  (v1 didn't have metadata)